specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-06-01/workspace.json

```diff
@@ -0,0 +1,1513 @@
+          "description": "Virtual Network profile",
+          "$ref": "#/definitions/VirtualNetworkProfile"
+        },
+        "connectivityEndpoints": {
+          "description": "Connectivity endpoints",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "type": "object"
+        },
+        "managedVirtualNetwork": {
+          "description": "Setting this to 'default' will ensure that all compute for this workspace is in a virtual network managed on behalf of the user.",
+          "type": "string"
+        },
+        "privateEndpointConnections": {
+          "description": "Private endpoint connections to the workspace",
+          "type": "array",
+          "items": {
+            "$ref": "../../../../common/v1/privateEndpointConnection.json#/definitions/PrivateEndpointConnection"
+          }
+        },
+        "encryption": {
+          "description": "The encryption details of the workspace",
+          "$ref": "#/definitions/EncryptionDetails"
+        },
+        "workspaceUID": {
+          "description": "The workspace unique identifier",
+          "type": "string",
+          "readOnly": true,
+          "format": "uuid"
+        },
+        "extraProperties": {
+          "description": "Workspace level configs and feature flags",
+          "additionalProperties": {
+            "type": "object"
+          },
+          "type": "object",
+          "readOnly": true
+        },
+        "managedVirtualNetworkSettings": {
+          "description": "Managed Virtual Network Settings",
+          "$ref": "#/definitions/ManagedVirtualNetworkSettings"
+        },
+        "workspaceRepositoryConfiguration": {
+          "description": "Git integration settings",
+          "$ref": "#/definitions/WorkspaceRepositoryConfiguration"
+        },
+        "purviewConfiguration": {
+          "description": "Purview Configuration",
+          "$ref": "#/definitions/PurviewConfiguration"
+        },
+        "adlaResourceId": {
+          "description": "The ADLA resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "publicNetworkAccess": {
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "description": "Enable or Disable public network access to workspace",
+          "type": "string",
+          "x-ms-enum": {
+            "name": "WorkspacePublicNetworkAccess",
+            "modelAsString": true
+          }
+        },
+        "cspWorkspaceAdminProperties": {
+          "description": "Initial workspace AAD admin properties for a CSP subscription",
+          "$ref": "#/definitions/CspWorkspaceAdminProperties"
+        },
+        "settings": {
+          "description": "Workspace settings",
+          "additionalProperties": {
+            "type": "object"
+          },
+          "type": "object",
+          "readOnly": true
+        },
+        "azureADOnlyAuthentication": {
```

Comment:
how does this property on the workspace interact with the azureAdOnlyAuthentication child resource?  If I change this does the child resource change?  What about vice-versa?  If a child resource change changes this how do you notify downstream services like ARG that this resource has changed?

Reply:
azureADOnlyAuthentication is more of a workspace setttings, then a child resource. User can modify its value either through PUT workspace call or PUT azureADOnlyAuthentication call.

In any of the case, Synapse updates the setting in its metastore and make a necessary RP call to partner service to reflect the change.